freqtrade/optimize/optimize_reports.py

```diff
@@ -556,15 +556,15 @@ def text_table_add_metrics(strat_results: Dict) -> str:
             ('Backtesting to', strat_results['backtest_end']),
             ('Max open trades', strat_results['max_open_trades']),
             ('', ''),  # Empty line to improve readability
-            ('Total trades', strat_results['total_trades']),
+            ('Total/Daily Avg Trades', 
```

Comment:
```suggestion
            ('Total/Daily Avg Trades',
```

Flake8 can be ... picky - sometimes :laughing:

Reply:
PyCharm s "Reformat Code" would ve easily prevented this. I m happy that i made this mistake, a good reminder of that feature, thanks